util/gh-pages/script.js

```diff
@@ -579,6 +579,40 @@ function setTheme(theme, store) {
     }
 }
 
+function getVirtualKey(ev) {
+    if ("key" in ev && typeof ev.key !== "undefined") {
+        return ev.key;
+    }
+
+    const c = ev.charCode || ev.keyCode;
+    if (c === 27) {
+        return "Escape";
+    }
+    return String.fromCharCode(c);
+}
```

Comment:
I don't think we need the fallback looking at the [browser support](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key#browser_compatibility)

Reply:
Fair enough. Removing it.